Gemfile

```diff
@@ -6,4 +6,4 @@ gemspec
 gem "jekyll", ENV["JEKYLL_VERSION"] ? "~> #{ENV["JEKYLL_VERSION"]}" : ">= 4.0"
 gem "minima"
 
-gem "sass-embedded", "~> 0.9.1" if RUBY_VERSION >= "2.6.0"
+gem "sass-embedded", "~> 0.19.0" if RUBY_VERSION >= "2.6.0"
```

Comment:
If the overall API (of all libraries involved) can be considered to be stable, do we need this constraint instead of the flexible `~> 0.19` ?

Reply:
The new API and all its dependencies (dart-sass-embedded & google-protobuf) are stable. I am actually planning to release 1.0.0 for the sass-embedded gem with legacy API removed after this PR is merged.

@ashmaroli Do you want to just remove this constraint? Or, maybe set it to `< 2`?

Reply:
What has this PR got that'd block you from releasing v1.0?

Reply:
The current master version is being used by a few people that is relying on the legacy API which is to be removed in 1.0.

I know it's their own risks using unreleased version, but I don't want to break them :-)

Reply:
Or we can: 
1. Leave it as it in this PR.
2. Merge this PR.
3. Release sass-embedded v1.0.
4. Send another PR to update Gemfile to require `~> 1.0`.
@ashmaroli What do you think?

Reply:
Go ahead and release sass-embedded v1.0 when ready. Update this PR after that to require just 1.x.
Since the version directive is in the Gemfile (instead of the plugin's gemspec), there's isn't going to be any difference for the end-user when this gets merged.

Reply:
The Gemfile is local to this repo, which it is not part of gemspec. If end users do not declare the version range and do not commit a Gemfile.lock file in their project, it will be open range and cause problem.

Reply:
@ashmaroli 1.0.0 is released and PR updated.